test/functional/observer_controller_test.rb

```diff
@@ -2446,4 +2446,19 @@ def test_download_observation_index
     assert_no_flash
     assert_response(:success)
   end
+
+  def test_normal_permissions
+    get :intro
+    assert_equal(200, @response.status)
```

Comment:
Is it any better to use `assert_response` instead of `assert_equal`?  (Error messages might be more specific; with some symbols as argument to `assert response`, the assertion is more flexible.  See http://api.rubyonrails.org/classes/ActionDispatch/Assertions/ResponseAssertions.html#method-i-assert_response.)
So instead of `assert_equal(200, @response.status)`, consider
  `assert_response(200)` or
  `assert_response(:success)`  (Status code 200-299)
And instead of  `assert_equal(403, @response.status)`
  `assert_response(403)` or
  `assert_response(:Forbidden)`  (same effect, but intent might be more transparent)